src/Agent.Plugins/Artifact/PipelineArtifactConstants.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace Agent.Plugins
```

Comment:
Not related to line: I suggest putting the changes behind a feature flag

Reply:
I did think about putting this behind a FF, however - since we will be controlling the default domain on server side (which won't be changed anytime soon) and a previous feature to get default chunk size form the same API is already in prod - this is a relatively safe change.